documentation/modules/auxiliary/scanner/dcerpc/hidden.md

```diff
@@ -0,0 +1,76 @@
+## Description
+
+The hidden scanner connects to a given range of IP addresses and try to locate any RPC services that are not listed in the Endpoint Mapper and determine if anonymous access to the service is allowed.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/dcerpc/hidden```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set THREADS [number of threads]```
+4. Do: ```run```
+
+As you can see, there are not many options to configure so we will just point it at some targets and let it run.
+
+## Scenarios
+
+**Running the scanner**
```

Comment:
Lets change this line to 

```
### Example Windows 2003, and Windows 7 Targets
```
(assuming the targets are the same from the previous markdown

Reply:
Yeah, but RPC service is there for linux also. Won't it be more accurate to keep it generic?

Reply:
sure you can leave it then